server/src/api/contributor/content-types/contributor/lifecycles.ts

```diff
@@ -0,0 +1,13 @@
+export default {
+  beforeCreate(event) {
```

Comment:
maybe abstract this and put into a helper or something similar? to not repeat yourself

Reply:
I mean whole lifecicle TS